lib/super_digit.rb

```diff
@@ -1,15 +1,18 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(log n)
+# Space Complexity - O(log n)
 def super_digit(n)
```

Comment:
Since you're summing and doing the recursion inside the sum.  This is O( (log n)^2)